tox.ini

```diff
@@ -0,0 +1,77 @@
+[tox]
+envlist = py37
+isolated_build = true
+skipsdist=True
+
+toxworkdir={toxinidir}/.tox
```

Comment:
I believe the default is already this value (`toxworkdir={toxinidir}/.tox(PATH)`)
Do we need to add the value?

Reply:
Just an explicit declaration for our requirements and dev files to be located in a better manner. There were some issues if we were directly trying to import and execute them.

Reply:
Thanks for clarifying.